includes/class-freemius.php

```diff
@@ -17770,7 +17834,8 @@ private function install_with_new_user(
             $install_public_key,
             $install_secret_key,
             $redirect = true,
-            $auto_install = false
+            $auto_install = false,
+            $is_diagnostic_tracking_allowed = null

```

Comment:
@msladu I suggest moving the new param to appear right after `$is_extensions_tracking_allowed` to logically appear next to the other customer permission (since it's a private method there's no need to think about backward compatibility here). Obviously, you also need to remember to update the calls.

This comment is also applicable to `install_many_pending_with_user()` and `install_many_with_new_user()`.